### PR TITLE
Test multisite

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Added framework for the archaeology multisite.

# Needed By (Date)
- ASAP

# Urgency
- High

# Steps to Test

1. Manual review.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: docroot/sites/default/settings.php [hunk 1]
```diff
@@ -790,10 +790,6 @@
 #   include $app_root . '/' . $site_path . '/settings.local.php';
 # }
 
-if (file_exists('/var/www/site-php')) {
-  require '/var/www/site-php/swshumsci/swshumsci-settings.inc';
-}
-
 require DRUPAL_ROOT . "/../vendor/acquia/blt/settings/blt.settings.php";
```
[the remark this answers is not in the document]
Comment (pookmish) [reply docroot/sites/default/settings.php]: it is what has the database credentials on acquia's servers. I guess multi-site install have to have a snippet like that pointing to the file for its respective database settings. But default doesn't need it and defaults to the normal credentials.

File: docroot/modules/humsci/hs_paragraph_types/modules/hs_hero_image/config/install/responsive_image.styles.hero_image.yml [hunk 2]
```diff
@@ -5,42 +5,38 @@ dependencies:
...
-  theme:
-    - stanford_basic
-_core:
-  default_config_hash: _ldYlaDS02eaxI53rCam6aU1VEhTWvjxNE4ZsLrd9Xo
 id: hero_image
 label: 'Hero image'
 image_style_mappings:
   -
-    breakpoint_id: stanford_basic.lg
+    breakpoint_id: stanford_media.lg
```
Comment (sherakama) [inline docroot/modules/humsci/hs_paragraph_types/modules/hs_hero_image/config/install/responsive_image.styles.hero_image.yml]: New dependency on stanford_media that isn't captured?

File: docroot/profiles/humsci/su_humsci_profile/config/install/block.block.su_humsci_theme_search.yml [hunk 3]
```diff
@@ -15,5 +15,5 @@ settings:
   id: search_form_block
   label: Search
   provider: search
-  label_display: visible
+  label_display: '-1'
```
Comment (sherakama) [inline docroot/profiles/humsci/su_humsci_profile/config/install/block.block.su_humsci_theme_search.yml]: -1 string?

Comment (pookmish) [reply docroot/profiles/humsci/su_humsci_profile/config/install/block.block.su_humsci_theme_search.yml]: thats how it exported 🤷‍♂️

File: docroot/modules/humsci/hs_paragraph_types/modules/hs_hero_image/config/install/responsive_image.styles.hero_image.yml (hunk 2 above)
Comment (pookmish) [reply docroot/modules/humsci/hs_paragraph_types/modules/hs_hero_image/config/install/responsive_image.styles.hero_image.yml]: https://github.com/SU-HSDO/suhumsci/blob/develop/docroot/modules/humsci/hs_paragraph_types/modules/hs_hero_image/hs_hero_image.info.yml#L11

File: docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.info.yml [hunk 4]
```diff
@@ -27,7 +27,6 @@ dependencies:
   - ckeditor
   - color_field
   - components
-  - config_readonly
```
Comment (sherakama) [inline docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.info.yml]: :( no more dependency?

Comment (sherakama) [reply docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.info.yml]: Oh I see why.

Comment (pookmish) [reply docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.info.yml]: during install it causes issues. once we get the hs_config_readonly i plan to add that as a dependency after installation. https://github.com/SU-HSDO/suhumsci/blob/test-multisite/docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install#L57

File: docroot/sites/archaeology/settings.php [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+require DRUPAL_ROOT . "/sites/default/settings.php";
+
+if (file_exists('/var/www/site-php')) {
```
Comment (sherakama) [inline docroot/sites/archaeology/settings.php]: This seems like a manual process that could be made more dynamic. I suppose you could scan that directory for settings.inc files and include them all. This is good for now.

File: docroot/modules/humsci/hs_paragraph_types/modules/hs_hero_image/config/install/responsive_image.styles.hero_image.yml (hunk 2 above)
Comment (sherakama) [reply docroot/modules/humsci/hs_paragraph_types/modules/hs_hero_image/config/install/responsive_image.styles.hero_image.yml]: Yeah, but this is a different config and has it's own dependencies list. It is missing.

Comment (pookmish) [reply docroot/modules/humsci/hs_paragraph_types/modules/hs_hero_image/config/install/responsive_image.styles.hero_image.yml]: 💥 got it 👍 https://github.com/SU-HSDO/suhumsci/pull/14/commits/9e899f9ff15744219b4d2be75f75aebccf07355e

**Pull request merged**